adafruit_button/button_base.py

```diff
@@ -72,6 +73,7 @@ def __init__(
         self._label_color = label_color
         self._label_font = label_font
         self._selected_label = _check_color(selected_label)
+        self._label_scale = label_scale or 1
```

Comment:
This is passed in to `Label` which wouldn't like a `None` value for scale, so this defaults the value to 1. I prefer for optional parameters to default to None, that way callers can pass in a None and get default behavior. But if it's desirable to set the default on the parameter to 1 as `Label` does it, I can change it.